### PR TITLE
📚 DOC: Add introduction sections to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by chrisjsewell**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,24 +23,41 @@ Thank you for you interest in contributing ✨
...
+
+This project and everyone participating in it is governed by the [EBP Code of Conduct](https://github.com/executablebooks/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [executablebooks@gmail.com](mailto:executablebooks@gmail.com).
+
+## I don't want to read this whole thing I just have a question!!!
+
+We have started trialing a GitHub discussion board where the community can chime in with helpful advice if you have questions:
+
+- https://github.com/executablebooks/meta/discussions
+
+## What should I know before I get started?
```
Comment (chrisjsewell) [inline CONTRIBUTING.md:43]:
```suggestion
## What should I know before I get started?

For EBP's overarching goals and principles, see: <https://executablebooks.org>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,24 +23,43 @@ Thank you for you interest in contributing ✨
...
 
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [EBP Code of Conduct](https://github.com/executablebooks/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [executablebooks@gmail.com](mailto:executablebooks@gmail.com).
+
+## I don't want to read this whole thing I just have a question!!!
+
+We have started trialing a GitHub discussion board where the community can chime in with helpful advice if you have questions:
+
+- https://github.com/executablebooks/meta/discussions
```
Comment (chrisjsewell) [inline CONTRIBUTING.md]:
```suggestion
We have started trialing a GitHub discussion board where the community can chime in with helpful advice if you have questions: <https://github.com/executablebooks/meta/discussions>

Additionally, if you would like to see a new feature implemented, see our [Feature Voting page](https://executablebooks.org/en/latest/feature-vote.html).
```

**Comment by chrisjsewell** [comment]
@choldgraf nudge to review while you're active

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,24 +23,43 @@ Thank you for you interest in contributing ✨
...
+- [jupyter-cache](https://github.com/executablebooks/jupyter-cache) is used by MyST-NB to execute notebooks and cache their results, such that they are only re-excuted during documentation builds when code cells change.
+- [sphinx-book-theme](https://github.com/executablebooks/sphinx-book-theme) is a sphinx HTML theme, designed to be optimal for the presentation of executable books.
+- [sphinx-copybutton](https://github.com/executablebooks/sphinx-copybutton), [sphinx-togglebutton](https://github.com/executablebooks/sphinx-togglebutton), [sphinx-panels](https://github.com/executablebooks/sphinx-panels) and [sphinx-thebe](https://github.com/executablebooks/sphinx-thebe) provide sphinx extensions to allow the inclusion of special features in the documentation.
+- [jupyter-book](https://github.com/executablebooks/jupyter-book) provides a user-friendly interface for building beautiful, publication-quality books and documents, utlising the above components.
+- [myst-language-support](https://github.com/executablebooks/myst-language-support) provides  aTextmate grammar, and VS Code extension, for editing MyST markdown.
```
Comment (choldgraf) [inline CONTRIBUTING.md:59]: this whole section makes me super impressed with all of the work that we have accomplished in 6 months, during a global pandemic, and all with <=  50% time to work on this.

**Comment by choldgraf** [comment]
This looks good to me 👍 are you auto-generating that TOC field somehow? Or just manually adding the links? Either way I like it!

**Comment by chrisjsewell** [comment]
Well originally with https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#table-of-contents

File: CONTRIBUTING.md (hunk 3 above)
Comment (chrisjsewell) [reply CONTRIBUTING.md:59]: certainly a few more commits than on docutils 😬 😆

**Pull request merged**